src/Connection/ConnectionInterface.php

```diff
@@ -78,4 +79,14 @@ public function getTableSchema(string $name, $refresh = false): ?TableSchema;
      * @return void
      */
     public function setEnableSlaves(bool $value): void;
+
+    public function isLoggingEnabled(): bool;
```

Comment:
I think it would be better named:

```php
public function isLogging(): bool;
```

Reply:
This feature was previously.
If renamed, its meaning can be understood in two ways:
   - The component supports logging.
   - The component has logging enabled.